app/scripts/controllers/build.js

```diff
@@ -94,15 +110,19 @@ angular.module('openshiftConsole')
       $scope.alerts["load"] = {
         type: "error",
         message: "The build details could not be loaded.",
-        details: "Reason: " + $filter('getErrorDetails')(e)
+        details: "Reason: " + $filter("getErrorDetails")(e)
       };
     };
 
     var buildConfigResolved = function(buildConfig, action) {
       if (action === "DELETED") {
         $scope.alerts["deleted"] = {
           type: "warning",
-          message: "Build configuration " + $scope.buildConfigName + " has been deleted."
+          message: (
+            "Build configuration " +
```

Comment:
while this makes it easier to see the parts that are being appended, i feel like it actually makes it harder to read the sentence its creating